tools/README.md

```diff
@@ -0,0 +1,17 @@
+# TaskTupleAwaiter tools
+
+these folders contain tools for the project
+
+Generator.CLI can be used like 
+
+ - ```generator d D:\ -v false -a 20 --format true```
+ - ```generator f D:\FileName.cs -v false -a 20 --format true```
+ 
+ d = directory
+ f = file
+
+ -v verify compilation
+ -a arity 
+ --format whether or not a formatter should be applied
```

Comment:
Markdown officially requires a double space at the end of a line if you actually want a line break. This is how GitHub renders .md files (unlike issue comments):

![image](https://user-images.githubusercontent.com/8040367/51803436-3df24c80-2223-11e9-8297-8480a8b83b7d.png)

Reply:
Thanks for letting me know. Fixed